src/Components/Web.JS/src/Virtualize.ts

```diff
@@ -57,7 +57,9 @@ function init(dotNetHelper: any, spacerBefore: HTMLElement, spacerAfter: HTMLEle
         return;
       }
 
-      const spacerSeparation = spacerAfter.offsetTop - (spacerBefore.offsetTop + spacerBefore.offsetHeight);
+      const spacerBeforeRect = spacerBefore.getBoundingClientRect();
+      const spacerAfterRect = spacerAfter.getBoundingClientRect();
+      const spacerSeparation = spacerAfterRect.top - spacerBeforeRect.bottom;
```

Comment:
Using `entry.boundingClientRect` here results in one spacer's rect being slightly more up-to-date than the other's, which is why both are being calculated here. I'm open to alternatives.

Reply:
This looks good to me.